### DISCUSSION TITLE
There is no instruction in the documentation how to get the Bearer Token

### DISCUSSION THREAD

**Discussion by matheusrossidev** [douglara/woofed-crm#327] [Q&A]

**Comment by douglas-gaspar-alves** [comment]
Estou com a mesma dificuldade!

**Comment by YukioArie** [comment]
Olá @douglas-gaspar-alves!
Vc consegue puxar essa informação pela pagina minha conta.
Segue o tutorial:

https://github.com/user-attachments/assets/a4469d06-8c1d-410a-bbab-734db3e2ff64
